BookingsApi/BookingsApi.IntegrationTests/Api/JusticeUsers/DeleteJusticeUserTests.cs

```diff
@@ -0,0 +1,127 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using BookingsApi.DAL;
+using BookingsApi.Domain;
+using BookingsApi.Domain.Enumerations;
+using BookingsApi.IntegrationTests.Helper;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using Testing.Common.Builders.Api;
+
+namespace BookingsApi.IntegrationTests.Api.JusticeUsers
+{
+    public class DeleteJusticeUserTests : ApiTest
+    {
+        private Guid? _justiceUserId;
+
+        [Test]
+        public async Task Should_delete_justice_user()
+        {
+            // Arrange
+            using var client = Application.CreateClient();
+            var justiceUserToDelete = await SeedJusticeUser("api_test_delete_justice_user@test.com");
+
+            // Act
+            var result = await client.DeleteAsync(
+                ApiUriFactory.JusticeUserEndpoints.DeleteJusticeUser(justiceUserToDelete.Id));
+            
+            // Assert
+            result.IsSuccessStatusCode.Should().BeTrue();
+            result.StatusCode.Should().Be(HttpStatusCode.NoContent);
+
+            await using var db = new BookingsDbContext(BookingsDbContextOptions);
+            var justiceUser = db.JusticeUsers.FirstOrDefault(x => x.Id == justiceUserToDelete.Id);
```

Comment:
Could you check for working and nonavailable hours too please

Reply:
this is covered in the [command handler tests](https://github.com/hmcts/vh-bookings-api/pull/595/files#diff-d14cb9331548282f945464e69cf026873d43dbed05589a6cfc2d1b283b5104a7). Based on the AddJusticeUser example, I thought the API tests were meant to be more of a high level test for the request/response. But I can copy the test here as well, if we think it's worth it